components/MarkdownTextBlock/src/MarkdownUIBuilder.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.Labs.WinUI.MarkdownTextBlock.Renderers;
+using CommunityToolkit.Labs.WinUI.MarkdownTextBlock.TextElements;
+using Markdig;
+
+namespace CommunityToolkit.Labs.WinUI.MarkdownTextBlock;
+
+public static class MarkdownUIBuilder
+{
+    public static UIElement Build(MarkdownConfig config)
+    {
+        if (config == null) throw new ArgumentNullException(nameof(config));
+        var pipeline = new MarkdownPipelineBuilder()
+            .UseEmphasisExtras()
+            .UseAutoLinks()
+            .UseTaskLists()
+            .UsePipeTables()
+            .Build();
```

Comment:
This is MarkDig configuration stuff, eh? Later, we probably want to think about how we can both cache this (for reducing overhead of re-rendering) as well as exposing this for additional settings by a developer.

Reply:
Yes agreed. There's a lot to think about in this area. I'll work on a dev design doc after this PR.